src/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ list(APPEND predict_public_header
   ${LIBPREDICT_INCLUDE_DIR}/predict/orbit.h
   ${LIBPREDICT_INCLUDE_DIR}/predict/observer.h
   ${LIBPREDICT_INCLUDE_DIR}/predict/tle.h
+  ${LIBPREDICT_INCLUDE_DIR}/predict/juliandate.h
+  ${LIBPREDICT_INCLUDE_DIR}/predict/predict.h
```

Comment:
I'm not too happy about adding more header files to the interface. We could just put everything in predict.h and include that everywhere, or split into two: typedefs/structs and functions.